src/meta_schedule/tune_context.cc

```diff
@@ -70,6 +70,82 @@ void TuneContextNode::Initialize() {
   }
 }
 
+void TuneContextNode::_GenerateMeasureCandidates() {
+  ICHECK(this->search_strategy.defined());
+  this->measure_candidates = this->search_strategy.value()->GenerateMeasureCandidates();
+}
+
+void TuneContextNode::_SendToBuilder(const Builder& builder) {
+  Array<MeasureCandidate> candidates = this->measure_candidates.value();
```

Comment:
It might be helpful to add a function for manually adding `measure_candidates`, since sometimes the field is empty after running search algorithms.

Reply:
Thanks! That's a good point!

Reply:
added